### PR TITLE
DB/Creature: [NPC] Rabine Saturna missing text Fixes Issue #1454

### PR DESCRIPTION
**Pull request by CookieMonsterDev**
DB/Creature: [NPC] Rabine Saturna missing text Fixes Issue #1454

##### CHANGES PROPOSED:

-  Insert SQL

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1454

##### TESTS PERFORMED:

Tested on local Windows realm. Text now works properly when talking to Creature.

##### HOW TO TEST THE CHANGES:

Execute SQL, enter game and .go creature 42332 and talk with him.

##### Target branch(es):

Master

### CODE REVIEW

**Comment by comix1988** [comment]
I tested this Pr and now the text window returns the dialogues.

good job @CookieMonsterDev

**Comment by CookieMonsterDev** [comment]
Thank you @comix1988 . Keep up the awesome bug reports!

**Comment by Maurowin** [comment]
its fine for me too

**Comment by FrancescoBorzi** [comment]
good job @CookieMonsterDev and thanks a lot @comix1988 and @Maurowin for testing this!

File: data/sql/updates/pending_db_world/rev_1550020912374137400.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(11801,0,0,0,62,0,100,0,5768,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Rabine Saturna - On Gossip Option Selected - Close Gossip"),
+(11801,0,1,0,62,0,100,0,5768,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Rabine Saturna - On Gossip Option Selected - Say Line 0");
+
+DELETE FROM `creature_text` WHERE `CreatureID`=11801;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(11801,0,0,"Speak with me, $n - if you are willing, the Cenarion Circle would eagerly accept your generous offer of aid.  The threat posed by the Felvine of Eldre'Thalas is indeed dire.",12,0,100,273,0,0,9545,0,"Rabine Saturna");
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1550020912374137400.sql]: Add a new line at the end of the file, please :)

Comment (CookieMonsterDev) [reply data/sql/updates/pending_db_world/rev_1550020912374137400.sql]: I can't anymore. Don't have access :/, i'll add the lines for all future pushes tho.

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1550020912374137400.sql]: it looks like the empty line is there already

**Pull request merged**